CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.8.3] - 2023-07-27
```

Comment:
#42 modified a public trait, and thus changed the public API of this crate. The next release requires a minor version bump.